src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1415,25 +1507,59 @@ namespace WaterToAirHeatPumpSimple {
+                        CoolCapAtPeak = (rhoair * VolFlowRate * (MixEnth - SupEnth)) +
+                                        FanCoolLoad; // load on the cooling coil which includes ventilation load and fan heat
                         CoolCapAtPeak = max(0.0, CoolCapAtPeak);
                         MixWetBulb = Psychrometrics::PsyTwbFnTdbWPb(state, MixTemp, MixHumRat, state.dataEnvrn->StdBaroPress, RoutineName);
-                        ratioTWB = (MixWetBulb + 273.15) / 283.15;
-                        // rated condenser water inlet temperature of 85F
-                        ratioTS = (((85.0 - 32.0) / 1.8) + 273.15) / 283.15;
-                        TotCapTempModFac = CurveManager::CurveValue(state, simpleWatertoAirHP.TotalCoolCapCurveIndex, ratioTWB, ratioTS, 1.0, 1.0);
-
-                        if (TotCapTempModFac > 0.0) {
-                            RatedCapCoolTotalDes = CoolCapAtPeak / TotCapTempModFac;
+                        RatedMixWetBulb = simpleWatertoAirHP.RatedEntAirWetbulbTemp;
+                        // calculate temperatue ratio at design day peak conditions
+                        ratioTWB = (MixWetBulb + state.dataWaterToAirHeatPumpSimple->CelsiustoKelvin) / Tref;
+                        PltSizNum =
+                            PlantUtilities::MyPlantSizingIndex(state,
+                                                               "COIL:" + simpleWatertoAirHP.WatertoAirHPType + ":WATERTOAIRHEATPUMP:EQUATIONFIT",
+                                                               simpleWatertoAirHP.Name,
+                                                               simpleWatertoAirHP.WaterInletNodeNum,
+                                                               simpleWatertoAirHP.WaterOutletNodeNum,
+                                                               ErrorsFound,
+                                                               false);
+                        if (PltSizNum > 0) {
+                            DesignEntWaterTemp = state.dataSize->PlantSizData(PltSizNum).ExitTemp;
+                            ratioTS = (DesignEntWaterTemp + state.dataWaterToAirHeatPumpSimple->CelsiustoKelvin) / Tref;
+                        } else {
+                            ShowSevereError(state, "Autosizing of total cooling capacity requires a loop Sizing:Plant object");
+                            ShowContinueError(state, "Autosizing also requires physical connection to a plant or condenser loop.");
+                            ShowContinueError(state,
+                                              "Occurs in COIL:" + simpleWatertoAirHP.WatertoAirHPType +
+                                                  ":WATERTOAIRHEATPUMP:EQUATIONFIT Object=" + simpleWatertoAirHP.Name);
+                            ErrorsFound = true;
+                        }
+                        // calculate temperatue ratio at rated conditions
```

Comment:
Retrieve plant design leaving fluid temperature, and used it as design entering water temperature in place of the hard coded 85F.